src/RemoteRequest/WpHttpRemoteGetRequest.php

```diff
@@ -98,7 +99,7 @@ public function __construct( $ssl_verify = true, $timeout = self::DEFAULT_TIMEOU
 	 *
 	 * @param string $url URL to get.
 	 * @return Response Response for the executed request.
-	 * @throws FailedToGetFromRemoteUrl If retrieving the contents from the URL failed.
+	 * @throws FailedRemoteRequest If retrieving the contents from the URL failed.
```

Comment:
This change doesn't seem to be correct.

```suggestion
	 * @throws FailedToGetFromRemoteUrl If retrieving the contents from the URL failed.
```

Reply:
How so? `FailedToGetFromRemoteUrl` implements `FailedRemoteRequest`.